app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -26,10 +26,23 @@ class Bolt_Boltpay_ShippingController extends Mage_Core_Controller_Front_Action
     const CACHE_ESTIMATE_PREFIX = 'BOLT_PREFETCH_ESTIMATE_';
 
     /**
-     * @var Mage_Core_Model_Cache
+     * @var Mage_Core_Model_Cache  The Magento cache where the shipping and tax estimate is stored
      */
     protected $_cache;
 
+    /**
+     * @var Bolt_Boltpay_Model_ShippingAndTax  Object that performs shipping and tax business logic
+     */
+    protected $_shippingAndTaxModel;
```

Comment:
This variable is declared, used, but the initialization seems to be missing.

Reply:
resolved.